bin/quipucords-installer

```diff
@@ -120,14 +120,34 @@ main() {
     esac
   done
 
-  if [[ $# -ne 1 ]]; then
-    echo "Error: Exactly one command is required." 1>&2
+  if [[ $# -lt 1 ]]; then
+    echo "Error: Command is missing." 1>&2
     usage
   fi
 
   COMMAND="$1"
   shift
 
+  if [[ "${COMMAND}" == "upgrade" ]]; then
+    if [[ $# -eq 1 ]]; then
+      COMMAND_OPTION="${1}"
+      shift
+      if [[ "${COMMAND_OPTION}" == "--db" ]]; then
```

Comment:
echo says

```
upgrade [--db | --keep-db]
```

But it looks like `--keep-db` is not recognized. Instead, inconsistently documented `--skip-db` is recognized.

I think we should chose one as default and only recognize one flag that is opposite of default. I would probably remove all data on upgrade as default, and recognize `--keep-db` as opt-in way for users to keep the data. We might even go with `--experimental-keep-db` to signal there might be problems (until we are comfortable to release something to all customers).